lib/surface/compiler.ex

```diff
@@ -718,33 +716,80 @@ defmodule Surface.Compiler do
     :ok
   end
 
-  defp missing_slot(mod, slot_name, template_meta, parent_meta) do
+  defp missing_slot(module, slot_name, meta, _defined_slot_names, true) do
+    message = """
+    no slot `#{slot_name}` defined in the component `#{module}`\
```

Comment:
```suggestion
    no slot `#{slot_name}` defined in the component `#{inspect(module)}`\
```

Let's use `inspect` here to avoid messages showing the module with the `Elixir.` prefix. The same goes for all other messages.